src/sections/synths/SynthsInfo.tsx

```diff
@@ -1,17 +1,24 @@
 import styled from 'styled-components';
-import React, { ReactNode, useState } from 'react';
+import React, { ReactNode, useMemo, useState } from 'react';
 import media from 'styled-media-query';
 import { theme } from 'src/styles/theme';
 import { resetButtonCSS } from 'src/styles/common';
 import SynthCard from './SynthCard';
 import { SynthsProps } from 'pages/synths';
+import { Checkbox } from 'src/components/Checkbox/Checkbox';
 
-const SynthsInfo = ({ tokenList, exchangeInfo, dictionarySynthStatus, synths }: SynthsProps) => {
+const SynthsInfo = ({ tokenList, l1, l2 }: SynthsProps) => {
+	const [showL2, setShowL2] = useState(true);
 	const [synthCategory, setSynthCategory] = useState('all');
 	const filteredSynths: Record<string, ReactNode[]> = {
 		all: [],
 	};
 
+	const { synths, exchangeInfo, dictionarySynthStatus } = useMemo(
+		() => (showL2 ? l2 : l1),
+		[showL2, l1, l2]
+	);
```

Comment:
Dont think we need a memo here. It's should be faster (and simpler) to just 
`const { synths, exchangeInfo, dictionarySynthStatus } = showL2 ? l2 : l1`